app/routes/order/search_users.js

```diff
@@ -1,31 +1,41 @@
 import Ember from "ember";
-import AuthorizeRoute from './../authorize';
-import AjaxPromise from 'stock/utils/ajax-promise';
+import AuthorizeRoute from "./../authorize";
+import AjaxPromise from "stock/utils/ajax-promise";
 
 export default AuthorizeRoute.extend({
-
   model(params) {
     let orderId = params.order_id;
-    let order = this.store.peekRecord('designation', orderId) || this.store.findRecord('designation', orderId);
+    let order =
+      this.store.peekRecord("designation", orderId) ||
+      this.store.findRecord("designation", orderId);
     return Ember.RSVP.hash({
       order,
-      recentUsers: new AjaxPromise("/recent_users", "GET", this.session.get("authToken"))
+      recentUsers: new AjaxPromise(
+        "/recent_users",
+        "GET",
+        this.session.get("authToken")
+      )
     });
   },
 
-  setupController(controller, model){
+  setupController(controller, model) {
     this._super(controller, model);
     this.get("store").pushPayload(model.recentUsers);
     let users = [];
-    model.recentUsers.users.forEach( (user) => {
-      let userObject = this.store.peekRecord('user', user.id);
-      user.organisations_users_ids.forEach( (org_user_id) => {
+    model.recentUsers.users.forEach(user => {
+      let userObject = this.store.peekRecord("user", user.id);
+      user.organisations_users_ids.forEach(org_user_id => {
         let organisationsUser = controller.findOrganisationsUser(org_user_id);
         this.store.pushPayload(organisationsUser);
       });
       users.push(userObject);
     });
 
     controller.set("filteredResults", users.uniq());
+  },
+
+  deactivate() {
```

Comment:
I think you can probably user `resetController` 

```
resetController(ctrl) {
  ctrl.set("searchText", "");
}
```

to avoid `this.controllerFor("order.searchUsers");`

Reply:
changed `deactivate` with `resetController`.